docs/Troubleshooting.md

```diff
@@ -138,3 +138,59 @@ and is a JSON object generated by the Quilt build system.
 
     `$ aws cloudformation describe-stack-events --stack-name QUILT_STACK_NAME`
 
+## Collect logs to report a bug
+
+To expedite the resolution of any errors encountered while using
+Quilt, please capture the following logs and attach to an email
+to support@quiltdata.io:
+
+### Browser Network logs
+
+1. Go to the affected page in your Quilt Catalog.
+1. Open the browser Developer tools:
+    - Google Chrome: Press **F12**, **Ctrl+Shift+I** or from the
+    Chrome menu select **More tools > Developer tools**.
+1. Select the **Network** tab.
+1. Ensure the session is recorded:
+    - Google Chrome: Check the red button in the upper left corner is set to **Record**.
+1. Ensure **Preserve Log** is enabled.
+1. Click the **Clear** button to clear out existing logs.
+1. Perform the action that triggers the error (e.g. clicking the `Download package` button).
+1. Export the logs as HAR format.
+    - Google Chrome: **Ctrl + Click** anywhere on the grid of
+    network requests and select **Save all as HAR with content**.
+1. Save the HAR-formatted file to your localhost.
+    ![Save browser Network error logs as HAR content](imgs/troubleshooting-logs-browser.png)
+
+### Browser Console logs
+
+1. Go to the affected page in your Quilt Catalog.
+1. Open the browser Developer tools:
+    - Google Chrome: Press **F12**, **Ctrl+Shift+I** or from the
+    Chrome menu select **More tools > Developer tools**.
+1. Select the **Console** tab.
+1. Click the **Clear** button to clear out existing logs.
+1. Perform the action that triggers the error (e.g. clicking the `Download package` button).
+1. Export the logs.
+    - Google Chrome: **Ctrl + Click** anywhere on the grid of
+    network requests and select **Save as...**.
+1. Save the log file to your localhost.
```

Comment:
Please join this with the previous section since there's so much overlap. To make things easier you can remove **Clear** form both sections because more data is usually better.